core/src/avm2/object/xml_object.rs

```diff
@@ -261,6 +261,8 @@ impl<'gc> TObject<'gc> for XmlObject<'gc> {
         // FIXME - implement everything from E4X spec (XMLObject::getMultinameProperty in avmplus)
         let read = self.0.read();
 
+        let name = handle_input_multiname(name.clone(), activation);
```

Comment:
We either need to move this under the index check, or change the check. Because otherwise `xml["@0"]` behaves like `xml["0"]`.

Reply:
I moved the check